protocol/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "protocol"
+version = "0.1.0"
+edition = "2022"
```

Comment:
Latest edition is 2021 - https://doc.rust-lang.org/edition-guide/editions/index.html

Reply:
Got it！ Thank you very much！